.chloggen/split-exporters.yaml

```
@@ -0,0 +1,21 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: exporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Splitting otlp, otlphttp and logging exporters into their own modules"
+
+# One or more tracking issues or pull requests related to the change
+issues: [6343]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: |-
+  The import path for these exporters can now be access directly:
+  - `go.opentelemetry.io/collector/exporter/loggingexporter`
+  - `go.opentelemetry.io/collector/exporter/otlpexporter`
+  - `go.opentelemetry.io/collector/exporter/otlphttpexporter`
+
```

I think we should explicitly mention how to migrate builder configuration:

```suggestion
  
  If using these exporters, modify your Collector builder configuration to use `gomod` directly, such as:
  -  `gomod: go.opentelemetry.io/collector/exporter/otlpexporter v0.63.0`
```

Maybe even pin an issue with instructions about how to do this in this repository, or a small script to go from the old configuration to the new one

---

Yes, that makes sense :+1:

---

Issue pinned and description of the issue updated to include the instructions